example/provider-extensions/kyverno-policies/add-registry-to-osc.yaml

```diff
@@ -126,3 +128,39 @@ spec:
           - path: "/spec/units/{{elementIndex}}/dropIns"
             op: add
             value: [{"name":"start-seed-registry-cache.conf","content":"{{startDropIn|base64_decode(@)}}"}]
+  - name: modify-gardener-node-init-path
+    match:
+      all:
+      - resources:
+          kinds:
+          - OperatingSystemConfig
+    mutate:
+      foreach:
+      - list: "request.object.spec.files[]"
+        preconditions:
+          all:
+          - key: "{{ element.path  || '' }}"
+            operator: Equals
+            value: "/var/lib/gardener-node-agent/init.sh"
+        patchesJson6902: |-
+          - path: "/spec/files/{{elementIndex}}"
+            op: replace
+            value: {"content":{"inline":{"data":"{{base64_encode('{{replace_all('{{base64_decode(element.content.inline.data)}}','$tmp_dir/gardener-node-agent','$tmp_dir/ko-app/gardener-node-agent')}}')}}","encoding":"b64"}},"path":"/var/lib/gardener-node-agent/init.sh","permissions":493}
```

Comment:
I would prefer octal notation for permissions as it is more readable and common.